pgpool.conf

```diff
@@ -0,0 +1,750 @@
+# ----------------------------
```

Comment:
I'm very loathe to add a 750-line configuration file into the image unless there's a very good reason to do so. I don't even have time to read it. :)